pkg/daemon/ceph/client/mirror.go

```diff
@@ -211,7 +211,7 @@ func GetMirroredPoolImages(context *clusterd.Context, clusterInfo *ClusterInfo,
 }
 
 // GetPoolMirroringInfo  prints the pool mirroring information
-func GetPoolMirroringInfo(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) (*cephv1.PoolMirroringInfo, error) {
+func GetPoolRadosNamespaceMirroringInfo(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) (*cephv1.MirroringInfo, error) {
```

Comment:
Does `ceph mirror pool info` return status for rados namespace mirroring as well? I thought it was just for pools and there is a separate command for rados namespace mirroring status?

Reply:
We need to pass 
```
rbd mirror pool info poolName/radosNamespaceName
```
And it will work

Reply:
Ok, how about if we just refer to these as pool mirroring methods? And in the comments we can mention that the rados namespace can also be used if `pool/namespace` is passed to the `resourceName`. Rados namespaces are naturally inside pools, so it seems simplest and logical that way.
```suggestion
func GetPoolMirroringInfo(context *clusterd.Context, clusterInfo *ClusterInfo, resourceName string) (*cephv1.MirroringInfo, error) {
```

Reply:
Done